### ISSUE TITLE
Question : Customize Web UI

### ISSUE THREAD

**Issue by glasserre** [drakkan/sftpgo#829]
Hello.
It's possible to customize the Web UI (mainly Login page) to cjange colors, etc ... ?

**Comment by drakkan** [comment]
Hi,

currently you have to change the css/html templates by hand. I'm working on this and I think I'll push this change before the end of the week, these new configuration options will be available:

```
"branding": {
          "web_admin": {
            "name": "",
            "short_name": "",
            "favicon_path": "",
            "logo_path": "",
            "login_image_path": "",
            "disclaimer_name": "",
            "disclaimer_path": "",
            "extra_css": []
          },
          "web_client": {
            "name": "",
            "short_name": "",
            "favicon_path": "",
            "logo_path": "",
            "login_image_path": "",
            "disclaimer_name": "",
            "disclaimer_path": "",
            "extra_css": []
          }
        }
```

so you can customize the application name, logo, favicon etc. add a disclaimer on the login page and provide custom css files to customize the colors

**Comment by glasserre** [comment]
Perfect, will be enough for my need.
For the web client login page, will be great to have an option to hide Web admin link or hide it by default.
Thanks for your quick answer 👍

**Comment by glasserre** [comment]
Thanks ! :)

**Comment by glasserre** [comment]
Hello. 
For my knowledge, where are store css/html templates files ? (i'm on ubuntu).

**Comment by drakkan** [comment]
Hi,

take a look at the configuration file: `/etc/sftpgo/sftpgo.json`, by default they are in `/usr/share/sftpgo`

P.S. please note that this feature is not included in tha v2.2.3 tag, you need to download a recent build artifact from the "Actions" page

**Comment by dw72** [comment]
It would be nice to have the ability to hide the version string on the Login page.
